[2023-04-01 15:04:40,773] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-01T15:04:35.487776+00:00 [queued]>
[2023-04-01 15:04:40,806] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-01T15:04:35.487776+00:00 [queued]>
[2023-04-01 15:04:40,808] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-01 15:04:40,809] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-04-01 15:04:40,810] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-01 15:04:40,840] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2023-04-01 15:04:35.487776+00:00
[2023-04-01 15:04:40,850] {standard_task_runner.py:52} INFO - Started process 230 to run task
[2023-04-01 15:04:40,857] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2023-04-01T15:04:35.487776+00:00', '--job-id', '14', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpm9jm8shb', '--error-file', '/tmp/tmp98h6z0so']
[2023-04-01 15:04:40,919] {standard_task_runner.py:80} INFO - Job 14: Subtask run_after_loop
[2023-04-01 15:04:41,101] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-01T15:04:35.487776+00:00 [running]> on host 94291ec70be5
[2023-04-01 15:04:41,237] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2023-04-01T15:04:35.487776+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-01T15:04:35.487776+00:00
[2023-04-01 15:04:41,251] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-04-01 15:04:41,286] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2023-04-01 15:04:41,346] {subprocess.py:85} INFO - Output:
[2023-04-01 15:04:41,349] {subprocess.py:92} INFO - 1
[2023-04-01 15:04:41,350] {subprocess.py:96} INFO - Command exited with return code 0
[2023-04-01 15:04:41,414] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20230401T150435, start_date=20230401T150440, end_date=20230401T150441
[2023-04-01 15:04:41,468] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-01 15:04:41,516] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
